python/pyspark/ml/tuning.py

```diff
@@ -75,6 +76,15 @@
 ]
 
 
+_SPARKML_TUNING_TEMP_DFS_PATH = "SPARKML_TUNING_TEMP_DFS_PATH"
```

Comment:
what about a new parameter instead of this env variable?

Reply:
Spark config is not available in some circumstances. so I suggest to use environmental variable to config.